.nvmrc

```diff
@@ -0,0 +1 @@
+v12.20.0
```

Comment:
This is great.  
What do you want to do about the CI code?  Right now it's installing '12.x', and i don't think nvmrc is driving anything there.
We could be explicit by changing the version github is installing in the workflow, or we could add 'nvm use' in deploy.sh and destroy.sh
Actually we probably need to do both.
Anyways, if you agree we should lock the CI system to what dev is using, we can do something about it.

Reply:
@macrael I pushed a bit to resolve this c3bba8bd66507921385c240234a286c7ae356ef1

Reply:
@macrael I pushed something to keep the CI system on the right version. see: 4efc832